### PR TITLE
[ci skip] Updated Node and Node Registration TPL

### PR DESCRIPTION
**Pull request by sauveergoel**
Signed-off-by: sauveergoel <sauveer.goel@accenture.com>

**Changelog**
- Updated Node and Node Registration TPL

**Reviewed by**
@sownak 

**Linked issue**
#976

### CODE REVIEW

File: platforms/corda-ent/configuration/roles/helm_component/templates/node.tpl [hunk 1]
```diff
@@ -56,7 +56,15 @@ spec:
...
       configPath: etc
-      cordaJarMx: 1524
+      cordaJar:
+        memorySize: 1524
+        unit: M
+      volume:
+        baseDir: /opt/corda/base
+      pod:
+        resources:
+          limits: 2056M
```
Comment (sownak) [inline platforms/corda-ent/configuration/roles/helm_component/templates/node.tpl:66]: bit too high. use similar to 1524 for now

File: platforms/corda-ent/configuration/roles/helm_component/templates/node_registration.tpl [hunk 2]
```diff
@@ -53,7 +53,15 @@ spec:
...
       configPath: etc
-      cordaJarMx: 1524
+      cordaJar:
+        memorySize: 1524
+        unit: M
+      volume:
+        baseDir: /opt/corda/base
+      pod:
+        resources:
+          limits: 2056M
```
Comment (sownak) [inline platforms/corda-ent/configuration/roles/helm_component/templates/node_registration.tpl:63]: same as above

File: platforms/corda-ent/configuration/roles/helm_component/templates/node.tpl (hunk 1 above)
Comment (sauveergoel) [reply platforms/corda-ent/configuration/roles/helm_component/templates/node.tpl]: done changes

File: platforms/corda-ent/configuration/roles/helm_component/templates/node_registration.tpl (hunk 2 above)
Comment (sauveergoel) [reply platforms/corda-ent/configuration/roles/helm_component/templates/node_registration.tpl]: done changes

File: platforms/corda-ent/configuration/roles/helm_component/templates/node.tpl [hunk 3]
```diff
@@ -56,7 +56,15 @@ spec:
         baseDir: /opt/corda
       jarPath: bin
       configPath: etc
-      cordaJarMx: 1524
+      cordaJar:
+        memorySize: 1524
+        unit: M
+      volume:
+        baseDir: /opt/corda/base
```
Comment (suvajit-sarkar) [inline platforms/corda-ent/configuration/roles/helm_component/templates/node.tpl:63]: there are two volume defined in this tpl ( line 56)

Comment (sauveergoel) [reply platforms/corda-ent/configuration/roles/helm_component/templates/node.tpl]: done chnages

**Review by suvajit-sarkar — APPROVED** [review]

**Pull request merged**